docs/HowToRun.md

```diff
@@ -3,14 +3,15 @@
 <!-- START doctoc generated TOC please keep comment here to allow auto update -->
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 
-# Table of Contents
```

Comment:
If you don't like this name, also update `doctoc` command in package.json.

Reply:
fixed: https://github.com/commercetools/commercetools-subscriptions/pull/22/commits/d88f27a251a86683aa8dc72537de510936e9ad4d